_includes/selector.html

```diff
@@ -572,7 +572,15 @@
                 if (this.active_packages.length === 1 && this.active_packages[0] === "Choose Specific Packages") {
                     return "Select at least one package.";
                 } else if (this.active_packages[0] === 'Standard') {
-                    var pkgs = this.additional_pip_packages.flatMap(libraryToPkg);
+                    var pkgs_to_show = this.additional_pip_packages;
+
+                    // Hide cuSpatial and cuProj for Nightly
+                    if (this.active_release === "Nightly") {
+                        pkgs_to_show = pkgs_to_show.filter(pkg => pkg !== "cuSpatial" && pkg !== "cuProj");
+                    }
```

Comment:
```suggestion
                    // possibly filter out any packages that are disabled in specific settings
                    pkgs_to_show = pkgs_to_show.filter(pkg => !disableUnsupportedPackage(pkg))
```

My JavaScript is not the best, but is it possible to do something like this everywhere, instead of repeating the literals `"cuSpatial"` and `"cuProj"` in so many places?

Putting those filters in place would make it easier to do this kind of thing in the future, and reduce the effort to update the selector again when 25.06 becomes the stable version.

Reply:
blegh ok I see, I'd missed that distinction. Thanks for considering it.